jira/1.0/modules/ROOT/pages/jira-connector-xml-maven.adoc

```diff
@@ -0,0 +1,72 @@
+= Microsoft Teams - XML and Maven Support - Mule 4
+
+Although you can manually code a Mule app in XML, it is more efficient to use Anypoint Studio:
```

Comment:
Remove this sentence.

Reply:
This sentence is in the template.

Reply:
I wonder why I thought it was out of place.